cookbook/controller/error_pages.rst

```diff
@@ -46,7 +46,7 @@ is used to render a Twig template to show the user.
 
 .. _cookbook-error-pages-by-status-code:
 
-This controller uses the HTTP status code, request format and the following
+This controller uses the HTTP status code, the request format and the following
```

Comment:
Imho it reads odd the way it is now. If we omitted "the" here, we should remove it before "following logic" too. Let's see what Ryan thinks about this.